checkIn/client/src/components/fields.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import styles from './fields.css';
```

Comment:
can also call like this,
require('./fields.css');
so u can directly call like id="idname" instead of  {styles.idname}

Reply:
@Supra-MS Oh is that right? Does that work with css modules? How is it able to then figure out which locally scoped ID name to apply?

Reply:
Ah nevermind, re-reading your comment, I realized what you meant. Using "require" enables direct access of an object. Thanks!